firefly_client/firefly_client.py

```diff
@@ -1346,11 +1313,10 @@ def set_pan(self, plot_id, x=None, y=None, coord='image'):
         """
 
         payload = {'plotId': plot_id}
-        if x and y:
-            if coord.startswith('image'):
-                payload.update({'centerPt': {'x': x, 'y': y, 'type': 'ImagePt'}})
-            else:
-                payload.update({'centerPt': {'x': x, 'y': y, 'type': 'J2000'}})
+        if coord.startswith('image'):
+            payload.update({'centerOnImage': 'true'})
+        elif x and y:
+            payload.update({'centerPt': {'x': x, 'y': y, 'type': 'WorldPt', 'cSys': coord}})
```

Comment:
Does the way that was documented not work? Using image with no x and y?

Reply:
It will still work that way, there was a problem with previous logic - it was only allowing `image` option to take effect if x and y were defined. But image should work independent of it, x and y only matter for world point.